doc/conf.py

```diff
@@ -263,5 +263,5 @@ def setup(app):
 
 config_block = {
     'varnish3': 'Varnish 3',
-    'varnish4': 'Varnish 4'
+    'varnish4': 'Varnish 4 & 5'
```

Comment:
This renames the tab to `Varnish 4 & 5`. When Varnish 6 comes along, we can hopefully drop Varnish 3 support to keep the tabs manageable. :wink: